connectors/uplynk/src/main/java/com/theoplayer/android/connector/uplynk/UplynkConfiguration.kt

```diff
@@ -0,0 +1,45 @@
+package com.theoplayer.android.connector.uplynk
+
+/**
+ * Describes the configuration of the Uplynk Media integration.
+ */
+enum class SkippedAdStrategy {
+    /**
+     * Plays none of the ad breaks skipped due to a seek.
+     */
+    PLAY_NONE,
+
+    /**
+     * Plays all the ad breaks skipped due to a seek.
+     */
+    PLAY_ALL,
+
+    /**
+     * Plays the last ad break skipped due to a seek.
+     */
+    PLAY_LAST
+}
+
+/**
+ * Describes the configuration of the Uplynk Media integration.
```

Comment:
AFAIK the name is just "Uplynk", not "Uplynk Media". (You might have confused this with "Verizon Media".)
```suggestion
 * Describes the configuration of the Uplynk integration.
```
You'll probably want to find-replace this throughout the rest of the code. 😉